### PR TITLE
[pt-br] Add tasks/tools/install-kubectl-windows.md

### PR DESCRIPTION
**Pull request by luisfroliveira**
added page in pt-br

tasks/tools/install-kubectl-windows.md

issue #48279

### CODE REVIEW

File: content/pt-br/docs/tasks/tools/install-kubectl-windows.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+     ```powershell
+     curl.exe -LO "https://dl.k8s.io/release/v{{< skew currentPatchVersion >}}/bin/windows/amd64/kubectl.exe"
+     ```
+
+   {{< note >}}
+   Para descobrir a versão estável mais recente (por exemplo, para scripts), veja
+   [https://dl.k8s.io/release/stable.txt](https://dl.k8s.io/release/stable.txt).
+   {{< /note >}}
+
+2. Validar o binario (opcional)
```
Comment (edsoncelio) [inline content/pt-br/docs/tasks/tools/install-kubectl-windows.md:39]:
```suggestion
2. Validar o binário (opcional)
```

**Review by stormqueen1990 — COMMENTED** [review]
/lgtm

**Comment by edsoncelio** [comment]
Thanks!
/approve

**Pull request merged**